lib/internal/Magento/Framework/View/Element/Html/Link/Current.php

```diff
@@ -87,7 +87,9 @@ private function getMca()
      */
     public function isCurrent()
     {
-        return $this->getCurrent() || $this->getUrl($this->getPath()) == $this->getUrl($this->getMca());
+        return $this->getCurrent() ||
+            preg_replace('/(\/index|(\/))+($|\/$)/', '', $this->getUrl($this->getPath()))
```

Comment:
Could you please move this regexp to private constant?

Reply:
Hello @swnsma  :  Does it satisfy the "Documentation of Backward Incompatible Changes" ? Thanks!
![image](https://user-images.githubusercontent.com/8234209/62766018-9e5c2380-babb-11e9-9d90-364f1d1052de.png)

Reply:
Hello @swnsma: It can not pass the Semantic Test if I add the new constant to the @api class.

Reply:
@edenduong, I think everything should be OK, if we add this constant as `private` for our internal purposes.

Reply:
@swnsma: I have changed it. Please check it again. Thanks.